tools/nemo_forced_aligner/README.md

```diff
@@ -0,0 +1,82 @@
+# NeMo Forced Aligner (NFA)
+
+A tool for doing Forced Alignment using Viterbi decoding of NeMo CTC-based models.
+
+## Usage example 
+
+``` bash
+python <path_to_NeMo>/NeMo/tools/nemo_forced_aligner/align.py \
+        pretrained_name="stt_en_citrinet_1024_gamma_0_25" \
+        model_downsample_factor=8 \
+        manifest_filepath=<path to manifest of utterances you want to align> \
+        output_dir=<path to where your ctm files will be saved>
+```
+
+## How do I use NeMo Forced Aligner?
+To use NFA, all you need to provide is a correct NeMo manifest (with `"audio_filepath"` and `"text"` fields).
+
+Call the `align.py` script, specifying the parameters as follows:
+
+* `pretrained_name`: string specifying the name of a CTC NeMo ASR model which will be automatically downloaded from NGC and used for generating the log-probs which we will use to do alignment. Any Quartznet, Citrinet, Conformer CTC model should work, in any language (only English has been tested so far). If `model_path` is specified, `pretrained_name` must not be specified.
+>Note: NFA can only use CTC models (not Transducer models) at the moment. If you want to transcribe a long audio file (longer than ~5-10 mins), do not use Conformer CTC model as that will likely give Out Of Memory errors.
```

Comment:
Doesnt it work on CPU? Maybe worth it to make the device an option.

Reply:
No, conformer_ctc_model.transcribe runs into OOM errors when the audio is ~5-10 mins long, even using a CPU. If you want it to work on longer audio you need to use a Citrinet model